### PR TITLE
GUI skeleton

### LINKED ISSUE #56: Sketch a draft of the GUI and build a skeleton for Library 3 (Library 3)
## User story
1. As a user
2. I want to have a good GUI that looks like a video editing tool
3. So that I can effectively and efficiently work with the software. 

This ticket depends on #49.  

## Acceptance criteria
* Sketch a draft of what the final GUI should look like 
* Plan the GUI understandable and intuitive 
* The goal is to create a skeleton that already contains the basic functionality of a video editing tool, e.g. selecting and loading videos, then showing the video

### PR DESCRIPTION
**Pull request by zino212**
Closes #56

### CODE REVIEW

**Comment by akriese** [comment]
Remember to leave out logic for now as the ticket focuses on building building screens with components (just skeletons) :)

**Comment by zino212** [comment]
I thought that the skeleton should already include basic functionality  [("e.g. selecting and loading videos, then showing the video")](https://github.com/orgs/amosproj/projects/27/views/2?pane=issue&itemId=44723599), so I tried to implement that.

**Comment by zino212** [comment]
This passage is part of both the skeleton and the simple GUI ticket so the borders are kind of blurred.
But you are probably right that the first ticket focuses on design and components and the second on functionality. @akriese

**Comment by zino212** [comment]
Regarding the sign-off: I tried rebasing and removing my mistakes, but I did not get it right. But I think these commits dont mess with the github statistics so maybe it is fine.

File: GUI/build.gradle.kts [hunk 1]
```diff
@@ -22,6 +22,7 @@ dependencies {
     // (in a separate module for demo project and in testMain).
     // With compose.desktop.common you will also lose @Preview functionality
     implementation(compose.desktop.currentOs)
+    implementation("org.bytedeco:javacv:1.5.9")
```
Comment (akriese) [inline GUI/build.gradle.kts:25]: Is this necessary?

**Review by akriese — APPROVED** [review]

**Pull request merged**